src/test/java/com/epam/tc/hw2/ex2/TestService.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.tc.hw2.ex2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+
+public class TestService {
```

Comment:
Why do you need this second class? Please merge base class into one

Reply:
I have merged class TestService into one